Shawntru/Rotten-Tomatillos
# Feature/search bar

## Is this a fix or a feature?
Feature

## What is the change?
Users now can search by a movie title and the homepage 

## What does it fix?
N/A
## Where should the reviewer start?
App.js line 12
App.test.js line 3
MovieCard.scss line 30
Movies.js line 6
Navbar.js line 4
Navbar.scss line 6
Navbar.test.js line 10

## How should this be tested?
On the homepage when user click on a search icon the search bar appears allowing users to type a name of the movie. The homepage will update the list of the movies based on provided search conditions

## Relevant Ticket(s)
#35
